hooks/pre_validations/12-clear_pulp_content.rb

```diff
@@ -1,16 +0,0 @@
-if app_value(:clear_pulp_content)
```

Comment:
This app value is still mentioned in `KATELLO.md` under data reset. Looks like we also forgot to update that from `--reset` to `--reset-data`. There's also more things that are out of date (`--foreman-proxy-content-parent-fqdn` as also removed and `--foreman-proxy-content-certs-tar` as also renamed (by me) a long time ago) so I'm fine with leaving that for a future PR. It even still mentions katello-installer.git.

Reply:
I think that whole Katello readme could use both some updates and some moving of things to the official Katello docs. I will take that on as a post merge task.